Language Tools + Time and Complexity Assignment/Duplicate_in_array.cpp

```diff
@@ -15,17 +15,17 @@ Sample Output:
 
 // arr - input array
 // size - size of array
-//ritikagarwal231
-#include<algorithm>
+//Mohak-Goel
```

Comment:
Can we remove this please?
Only the question in comment would be better. Name of the contributor will be available in annotations.